Remind_Device/Remind_Device.ino

```diff
@@ -193,3 +236,12 @@ void loop() {
   }
     
 }                                    // Go back to the beginning of the loop
+
+
+//===========================================
+//             Custom Functions
```

Comment:
If we end up with a lot of custom functions, it might be nice to have a header file for them. Depends on how much we end up having. This might be especially nice depending on how complicated the wave libraries get.